@stellar/typescript-wallet-sdk/test/e2e/README.md

```diff
@@ -0,0 +1,11 @@
+# How it works
+
+## browser.test.ts
+
+This test uses playwright to load the browser bundle file into a browser
```

Comment:
That's smart! Do you think we should add some minimal sanity chromium/firefox/safari tests in the future?

Reply:
ya that's a good idea 👍 

it was quick so I just added, added firefox and safari
https://github.com/stellar/typescript-wallet-sdk/pull/126/commits/df1c6406af711e42d584b32252dbf9d7422cd171

Reply:
hmm firefox actually is throwing a weird error, seems to be related with running CI/CD, since it works locally

I added a ticket to look into and add in another PR
https://stellarorg.atlassian.net/browse/WAL-1448

Reply:
Maybe you can try to use stable firefox, not nightly?

Reply:
just saw this, I think playwright might be using nightly as default? 🤔 

there's a [thread](https://github.com/microsoft/playwright/issues/6500) discussing this error, skimmed it and doesn't look like it's a quick fix. I'll look more into it